src/Normalizer/AutoMapperNormalizer.php

```diff
@@ -54,14 +54,15 @@ public function supportsNormalization(mixed $data, string $format = null): bool
         return $this->autoMapper->hasMapper($data::class, 'array');
     }
 
-    public function supportsDenormalization(mixed $data, string $type, string $format = null): bool
+    /** @phpstan-ignore-next-line */
+    public function supportsDenormalization(mixed $data, string $type, string $format = null, array $context = []): bool
     {
         return $this->autoMapper->hasMapper('array', $type);
     }
 
-    public function hasCacheableSupportsMethod(): bool
+    public function getSupportedTypes(?string $format): array
     {
-        return true;
+        return ['object' => false];
```

Comment:
```suggestion
        return ['object' => true];
```
?
we don't use context in `supports(De)Normalization()`